pubspec.yaml

```diff
@@ -51,7 +52,8 @@ dev_dependencies:
 
 # The following section is specific to Flutter.
 flutter:
-
+  assets:
+    - assets/
```

Comment:
here sometimes different kinds of assets (not only pictures, also icons, sometimes even sounds or some text/music files, etc.), so I'd recommend using either `/images` or `assets/images`. Here it's not critical, but just keep that in mind.